Parameterized-Remote-Trigger.iml

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think you should not include this `iml` file in your pull request.

Reply:
Removed from pull request.